src/dawn/Optimizer/StencilInstantiation.h

```diff
@@ -563,6 +566,13 @@ class StencilInstantiation : NonCopyable {
     return BoundaryConditionToExtentsMap_.find(stmt)->second;
   }
 
+  Array3i getFieldIDFromInitializedDimensionsMap(int FieldID) {
```

Comment:
also the name is misleading, here we are getting something like `dimension mask`, not even a map, 
and the getter indicates `getFieldID`